Pipfile

```diff
@@ -13,7 +13,7 @@ python_version = "3.6"
 [packages]
 
 cfenv = "==0.5.3"
-flask = "==0.12.2"
+flask = "*"
 flask-cors = "==3.0.3"
```

Comment:
Can we try removing the pin for all the packages and updating everything? This is what we have done for all the other repos, hopefully doesn't require to many fixes 🤞(if it does leave it)